<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1069</width>
    <height>711</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1069</width>
    <height>711</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1069</width>
    <height>711</height>
   </size>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>EasyMail</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../dove1.png</normaloff>../dove1.png</iconset>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>181</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QListWidget" name="listWidget">
      <item>
       <property name="text">
        <string>Compose</string>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="background">
        <brush brushstyle="NoBrush">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </property>
       <property name="foreground">
        <brush brushstyle="NoBrush">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../pencil2.png</normaloff>../pencil2.png</iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Inbox</string>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../inbox.png</normaloff>../inbox.png</iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Drafts</string>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../contacts.png</normaloff>../contacts.png</iconset>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>20</y>
     <width>821</width>
     <height>671</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="pageCompose">
       <widget class="QLabel" name="labelNewMail">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Microsoft YaHei Light</family>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>New Mail</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelTo">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Microsoft YaHei Light</family>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>To:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditTo">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>40</y>
          <width>721</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Microsoft YaHei UI</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhNone</set>
        </property>
       </widget>
       <widget class="QLabel" name="labelSubject">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Microsoft YaHei Light</family>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Subject:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditSubject">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>80</y>
          <width>721</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Microsoft YaHei UI</family>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
       <widget class="QTextEdit" name="textEdit">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>120</y>
          <width>801</width>
          <height>491</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Microsoft YaHei UI</family>
          <pointsize>10</pointsize>
         </font>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonSend">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>620</y>
          <width>71</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Microsoft YaHei UI</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonSave">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>620</y>
          <width>71</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Microsoft YaHei UI</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="pageInbox">
       <widget class="QListWidget" name="listWidget_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>281</width>
          <height>671</height>
         </rect>
        </property>
       </widget>
       <widget class="QScrollBar" name="verticalScrollBar">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>0</y>
          <width>21</width>
          <height>671</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
       <widget class="QTextBrowser" name="textBrowser">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>0</y>
          <width>541</width>
          <height>681</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="pageContacts">
       <widget class="QListWidget" name="listWidget_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>261</width>
          <height>671</height>
         </rect>
        </property>
       </widget>
       <widget class="QScrollBar" name="verticalScrollBar_2">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>0</y>
          <width>21</width>
          <height>671</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
       <widget class="QTextBrowser" name="textBrowser_2">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>0</y>
          <width>541</width>
          <height>671</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
